app/Http/Controllers/Blogs.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Blogs extends Controller
+{
+    public function index()
```

Comment:
Remember to follow op on these automatically generated PHPcs comments. In the exam, there might be a point deduction when we find them in your code.